src/main/java/org/opentripplanner/middleware/models/RelatedUser.java

```diff
@@ -0,0 +1,29 @@
+package org.opentripplanner.middleware.models;
+
+/** A related user is a companion or observer requested by a dependent. */
+public class RelatedUser {
+    public enum RelatedUserStatus {
+        PENDING, CONFIRMED, INVALID
+    }
+
+    public String email;
+    public RelatedUserStatus status = RelatedUserStatus.PENDING;
+    public String acceptKey;
+    public String nickName;
```

Comment:
The spell checker in MS Word says "nickname"...

Reply:
Updated in: https://github.com/ibi-group/otp-middleware/pull/256/commits/4a00bfa58c2f747e1f645b4b2b0a4739ac136b6f